packages/lodestar-params/src/presets/mainnet.ts

```diff
@@ -10,8 +10,18 @@ export const commit = "v1.0.0";
 export const phase0Yaml = mapValuesNumToString(mainnetYaml);
 export const phase1Yaml = mapValuesNumToString(phase1MainnetYaml);
 
+const phase0Params = createParams<IBeaconParams>(phase0Yaml, BeaconParams);
+
 export const params: IBeaconParams = {
-  ...createParams<IBeaconParams>(phase0Yaml, BeaconParams),
+  ...phase0Params,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  ALL_FORKS: [
+    {
+      epoch: 0,
+      currentVersion: 16777216,
```

Comment:
Could you add a note on where to get this number from?

Reply:
for conversation sake, I've converted array to number mannually to avoid bringing lodestar-utils here just for that.
I'll add todo to remove this since this should go in yaml config but didn't hit dev branch yet